cql3/util.hh

```diff
@@ -39,21 +39,45 @@ Result do_with_parser(const sstring_view& cql, Func&& f) {
     return std::move(*ret);
 }
 
-template<typename Range> // Range<cql3::relation_ptr>
+template<typename Range> // Range<expr::expression>
 sstring relations_to_where_clause(Range&& relations) {
-    auto expressions = relations | boost::adaptors::transformed(std::mem_fn(&relation::to_string));
+    auto expr_to_pretty_string = [](const expr::expression& e) -> sstring {
+        expr::expression::printer p {
+            .expr_to_print = e,
+            .debug_mode = false,
+        };
+        return fmt::format("{}", p);
```

Comment:
When debug_mode=false, can't you just use format("{}", e) directly?

Reply:
No, the default is printing in debug mode, because that's how it was before.